01_fib/pfannerj/main_test.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	mainout = &buf
+
+	main()
+
+	expected := `Fibonacci series starting...
+Fibonacci series completed...
+`
+	actual := buf.String()
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestFibOutputFibZero(t *testing.T) {
+	var buf bytes.Buffer
+	fibout = &buf
+
+	fib(0)
+
+	expected := ``
+	actual := buf.String()
+	if expected != actual {
+		t.Errorf("Unexpected output in fib(0)")
+	}
+}
+
```

Comment:
It would be nice to see a test for `fib(1)`. Boundary conditions are often where bugs hide. I'd suggest that you repurpose the `fib(9)` test since that's not really testing any more than the `fib(7)` test, particularly since you have a `fib(-9)` test too.

Reply:
Yeah, I agree that fib(1) would be a better to include another edge case.  Also noted your observation above regarding the comment style and the commits.. I did squash some and will do further squashing for the other Labs now that I know how to do it :)